packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/pages/home/index.jsx.hbs

```diff
@@ -61,7 +61,14 @@ const Home = () => {
     const {data: productSearchResult, isLoading} = useProductSearch({
         parameters: {
             refine: [`cgid=${HOME_SHOP_PRODUCTS_CATEGORY_ID}`, 'htype=master'],
-            expand: ['promotions', 'variations', 'prices', 'images', 'custom_properties'],
+            expand: [
```

Comment:
Based on the discussion with Nathan, the decision was not to add `availability` in the productSearch as it has less cache time. We are ok with displaying the unorderable swatch in the PLP.  May be for this PR we can just show the price as zero if it is not avialble in the API response

Reply:
It will still clickable and will be directed to PDP, if we don't use orderable property, what are other ways to disable it?